editor/plugins/spatial_editor_plugin.cpp

```diff
@@ -77,11 +77,11 @@ void SpatialEditorViewport::_update_camera(float p_interp_delta) {
 		camera->set_perspective(get_fov(), get_znear(), get_zfar());
 
 	//when not being manipulated, move softly
-	float free_orbit_inertia = EDITOR_DEF("editors/3d/free_orbit_inertia", 0.15);
-	float free_translation_inertia = EDITOR_DEF("editors/3d/free_translation_inertia", 0.15);
+	float free_orbit_inertia = EDITOR_DEF("editors/3d/orbit_inertia", 0.15);
+	float free_translation_inertia = EDITOR_DEF("editors/3d/translation_inertia", 0.15);
```

Comment:
especially this naming might not be liked..
but I could associate anything with `free` If that is the case for others too it might be better without it? idk